libraries/ESP8266SSDP/ESP8266SSDP.cpp

```diff
@@ -47,12 +47,10 @@ extern "C" {
 #include "include/UdpContext.h"
 //#define DEBUG_SSDP  Serial
 
-#define SSDP_INTERVAL     1200
 #define SSDP_PORT         1900
 #define SSDP_METHOD_SIZE  10
 #define SSDP_URI_SIZE     2
 #define SSDP_BUFFER_SIZE  64
-#define SSDP_MULTICAST_TTL 2
```

Comment:
I would leave `SSDP_INTERVAL` and `SSDP_MULTICAST_TTL` defined and used in member initializers for clarity (maybe also adding the unity: `SSDP_INTERVAL_SECONDS`)

Reply:
that'd defeat the purpose of the PR to only have initialization once in one place. I can rename the _interval member variable to _interval_s to keep the unit.

this define is used only in one place, adding this #define does not add make things easier (you need to look it up anyway when you come across reading the constructor). 

if thats a hard objection the only thing I'd do is then to remove ESP82666SSDP* completely from this cleanup, leaving it as it is now.

Reply:
What @d-a-v means is expose the defines, i.e.: move them to the header, maybe turn them into constexpr, and use the relevant ones in the direct member assignment instead of using literal numbers that don't convey meaning.

Reply:
BTW, if it is true that the new MDNS responder also handles SSDP, then this SSDP class will likely be pulled out.